scenarios/getting_started/python/getting_started.py

```diff
@@ -9,56 +16,89 @@ def on_connect(client, _userdata, _flags, rc):
     
     print("Connected to MQTT broker")
     topic = "sample/+"
+    # SUBSCRIBE
     (_subscribe_result, subscribe_mid) = client.subscribe("sample/+")
     print(f"Sending subscribe requestor topic \"{topic}\" with message id {subscribe_mid}")
 
 def on_subscribe(client, _userdata, mid, _granted_qos):    
     print(f"Subscribe for message id {mid} acknowledged by MQTT broker")
+    # PUBLISH
     topic = "sample/topic1"
     payload = "hello world!"
     publish_result = client.publish(topic, payload)
```

Comment:
This is likely inadvisable due to the thread model of Paho. I recommend instead triggering a notification of some kind (Condition, Event, etc.) and then responding to that notification elsewhere.

The contents of a callback are always invoked on the Paho thread. Invoking Paho on its own background thread likely could lead to some issues, and additionally, it just muddies the readability of the sample.